contracts/interfaces/IPeripheryPayments.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import {Currency} from "@uniswap/v4-core/contracts/types/Currency.sol";
+
+/// @title Periphery Payments
+/// @notice Functions to ease deposits and withdrawals of ETH
+interface IPeripheryPayments {
+    // TODO: figure out if we still need unwrapWETH9 from v3?
```

Comment:
hmmm good question. I think its more of an opinionated decision nowadays... I would lean towards not adding it? People can add liquidity in ETH and remove it in ETH, people can add liquidity in WETH and remove it in WETH. It feels opinionated now to allow the WETH people to remove it as ETH if we arent doing it the other way around?

Reply:
however we do like to be helpful if we think its a common usecase of course

Reply:
yeah agreed on not making it opinionated